### PR TITLE
rdbc-624

### PR DESCRIPTION
**Pull request by alxsabo**
https://issues.hibernatingrhinos.com/issue/RDBC-624/PHP-Document-Query-round-2

For one of the test in this issue it was required to solve:

https://issues.hibernatingrhinos.com/issue/RDBC-620/Support-session-interface-that-allow-us-to-convert-json-document-to-php-array

### CODE REVIEW

File: tests/Test/Client/Issues/RavenDB_13452Test/RavenDB_13452Test.php [hunk 1]
```diff
@@ -26,6 +26,14 @@ public function testCanModifyDictionaryWithPatch_Add(): void
 
             $session = $store->openSession();
             try {
+
+                $item = $session->load(null, "items/1");
+
+                print_r($item);
```
Comment (ml054) [inline tests/Test/Client/Issues/RavenDB_13452Test/RavenDB_13452Test.php]: remove?

File: tests/Test/Issues/RavenDB_9745Test/RavenDB_9745Test.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace tests\RavenDB\Test\Issues\RavenDB_9745Test;
+
+use tests\RavenDB\RemoteTestBase;
+
+class RavenDB_9745Test extends RemoteTestBase
+{
+//    public void explain() throws Exception {
```
Comment (ml054) [inline tests/Test/Issues/RavenDB_9745Test/RavenDB_9745Test.php]: ?

Comment (alxsabo) [reply tests/Test/Issues/RavenDB_9745Test/RavenDB_9745Test.php]: This test is not in the list of tests from current issue, but index from this test is used in RavenDB_10520Test, so I just add it here for consistency and comment it out

Comment (ml054) [reply tests/Test/Issues/RavenDB_9745Test/RavenDB_9745Test.php]: image

I can see it here?

Comment (alxsabo) [reply tests/Test/Issues/RavenDB_9745Test/RavenDB_9745Test.php]: My bad. It's implemented now.

**Pull request merged**